### PR TITLE
[Wallet] Edit session namespaces

### PR DESCRIPTION
**Pull request by flypaper0**
**What changed**
- replaced iconURL's because old doesn't load
- SessionDetails screen rewrited to SwiftUI
- Ability to delete namespace properties 

**Video** _(Sorry for quality, 10MB limit)_

https://user-images.githubusercontent.com/5384197/170071174-1a3e6546-634d-4d74-a130-89aa262d1dd2.mov

### CODE REVIEW

**Comment by MisterVants** [comment]
We could use some protocol abstraction to add a full preview to this view in the future.

File: Example/ExampleApp/SessionDetails/SessionDetailViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,140 @@
+import Foundation
+import Combine
+import WalletConnectAuth
+
+@MainActor
```
Comment (MisterVants) [inline Example/ExampleApp/SessionDetails/SessionDetailViewModel.swift:5]: Why does it need to be a `@MainActor`? Does it have something to do with setting a `@Published` var during an async call?

**Review by MisterVants — APPROVED** [review]

**Review by llbartekll — APPROVED** [review]

File: Example/ExampleApp/SessionDetails/SessionDetailViewModel.swift (hunk 1 above)
Comment (flypaper0) [reply Example/ExampleApp/SessionDetails/SessionDetailViewModel.swift:5]: Yes, ensure what UI update in main thread

**Comment by flypaper0** [comment]
> We could use some protocol abstraction to add a full preview to this view in the future.

Anyway we need to mock SessionDetailViewModel somehow to display screen. Or what you mean?

**Pull request merged**